packages/data-point/package.json

```diff
@@ -48,6 +48,7 @@
     "lib/"
   ],
   "dependencies": {
+    "@types/react": "^16.9.5",
```

Comment:
why is this needed? probably needs to be removed

Reply:
done